make in build/alpine produces error 137

I am trying to build `wire-server` on my MBP (MacOS Mojave 10.14.2) with Docker version 18.09.0, build 4d60db4. After a while I the build ends with error 137, with no further log messages, as per below: 
```
...
vector-algorithms-0.7.0.4: download
vector-algorithms-0.7.0.4: configure
vector-algorithms-0.7.0.4: build
The command '/bin/sh -c apk add --no-cache git ncurses &&     mkdir -p /src && cd /src &&     git clone -b develop https://github.com/wireapp/wire-server.git &&     cd wire-server &&     stack update &&     echo "allow-different-user: true"                                       >> /root/.stack/config.yaml &&     echo                                                                    >> /root/.stack/config.yaml &&     echo '# NB: do not touch following line!'                               >> /root/.stack/config.yaml &&     echo '# this image is used both for building docker images with the'    >> /root/.stack/config.yaml &&     echo '# integration tests (so they can be run on the ci) and for'       >> /root/.stack/config.yaml &&     echo '# interactive integration testing (with the working copy of the'  >> /root/.stack/config.yaml &&     echo '# host system mounted into the docker container).  in the latter' >> /root/.stack/config.yaml &&     echo '# use case, we want the docker container to write to its own'     >> /root/.stack/config.yaml &&     echo '# stack-work directory and not pollute the one on the host.'      >> /root/.stack/config.yaml &&     echo 'work-dir: .stack-docker'                                          >> /root/.stack/config.yaml &&     stack --work-dir .stack-docker-profile build --haddock --dependencies-only --profile haskell-src-exts &&     stack --work-dir .stack-docker         build --haddock --dependencies-only           haskell-src-exts &&     stack --work-dir .stack-docker-profile build --haddock --no-haddock-hyperlink-source --profile haskell-src-exts &&     stack --work-dir .stack-docker         build --haddock --no-haddock-hyperlink-source           haskell-src-exts &&     stack --work-dir .stack-docker-profile build --pedantic --haddock --test --no-run-tests --bench --no-run-benchmarks --dependencies-only --profile &&     stack --work-dir .stack-docker         build --pedantic --haddock --test --no-run-tests --bench --no-run-benchmarks --dependencies-only' returned a non-zero code: 137
make: *** [builder] Error 137
```

--- a/README.md
+++ b/README.md
@@ -112,7 +112,9 @@ If you wish to build your own docker images, you need [docker version >= 17.05](
 make docker-services
 ```
 
-will, eventually, have built a range of docker images. See the `Makefile`s and `Dockerfile`s, as well as [build/alpine/README.md](build/alpine/README.md) for details.
+will, eventually, have built a range of docker images. Make sure to [give Docker enough RAM](https://github.com/wireapp/wire-server/issues/562); if you see `make: *** [builder] Error 137`, it might be a sign that the build ran out of memory. You can also mix and match – e.g. pull the [`alpine-builder`](https://quay.io/repository/wire/alpine-builder?tab=tags) image and build the rest locally.
+
+See the `Makefile`s and `Dockerfile`s, as well as [build/alpine/README.md](build/alpine/README.md) for details.
 
 ### How to run integration tests
 

